consumer/http_v4_test.go

```diff
@@ -49,39 +50,39 @@ func TestHttpV4TypeSystem(t *testing.T) {
 			return nil
 		})
 	assert.Error(t, err)
+	if os.Getenv("SKIP_PLUGIN_PROTOBUF") != "1" {
+		dir, _ := os.Getwd()
+		path := fmt.Sprintf("%s/pact_plugin.proto", strings.ReplaceAll(dir, "\\", "/"))
```

Comment:
2 changes per test file plugins

1. all SKIP_PLUGIN_<PLUGIN_NAME>
2. replace backslashes with forward slashes in `PluginContents` for constructed path to proto file (should we handle in the protobuf plugin itself so save users having to do this manually on windows platforms)